terminal-widgets.cabal

```diff
@@ -61,7 +61,6 @@ library
         base >= 4.16 && < 5,
         exceptions,
         generic-lens,
-        lens,
```

Comment:
Why are we keeping `generic-lens`?

Reply:
`generic-lens` has a light footprint; `lens` is huge, but `generic-lens` is comparatively small. removing all of generic-lens would require much more work (e.g. the virtual fields are used in quite a few places). Removing `generic-lens` can be done in another PR (this is just about removing `lens`, which is the big offender).